Framework/Algorithms/test/MaxEntTest.h

```diff
@@ -128,9 +173,85 @@ class MaxEntTest : public CxxTest::TestSuite {
     TS_ASSERT_THROWS_ANYTHING(alg->execute());
   }
 
+  void test_bad_linear_adjustment() {
+
+    auto ws = WorkspaceCreationHelper::create2DWorkspace(5, 10);
+
+    IAlgorithm_sptr alg = AlgorithmManager::Instance().create("MaxEnt");
+    alg->initialize();
+    alg->setChild(true);
+    alg->setProperty("InputWorkspace", ws);
+    alg->setProperty("ComplexData", false);
+    alg->setPropertyValue("MaxIterations", "1");
+    alg->setProperty("DataLinearAdj", ws);
+    alg->setPropertyValue("ReconstructedImage", "image");
+    alg->setPropertyValue("ReconstructedData", "data");
+    alg->setPropertyValue("EvolChi", "evolChi");
+    alg->setPropertyValue("EvolAngle", "evolAngle");
+
+    TS_ASSERT_THROWS_ANYTHING(alg->execute());
```

Comment:
we can use `TS_ASSERT_THROWS(alg->execute(), ...) ` here, where ... is the type of error thrown.

Reply:
I've tried this but the algorithm class changes the error type to something else and so I doubt it would be useful.